portality/forms/application_processors.py

```diff
@@ -441,42 +441,41 @@ def finalise(self, account, save_target=True, email_alert=True):
                     self.add_alert('Sending the ready status to managing editors didn\'t work. Please quote this magic number when reporting the issue: ' + magic + ' . Thank you!')
                     app.logger.exception('Error sending ready status email to managing editors - ' + magic)
 
-    def _send_application_approved_email(self, journal_title, publisher_name, email, update_request=False):
+    def _send_application_approved_email(self, application, journal, owner, update_request=False):
         """Email the publisher when an application is accepted (it's here because it's too troublesome to factor out)"""
         url_root = request.url_root
         if url_root.endswith("/"):
             url_root = url_root[:-1]
 
-        to = [email]
+        to = [owner.email]
         fro = app.config.get('SYSTEM_EMAIL_FROM', 'feedback@doaj.org')
         if update_request:
             subject = app.config.get("SERVICE_NAME", "") + " - update request accepted"
         else:
             subject = app.config.get("SERVICE_NAME", "") + " - journal accepted"
-        publisher_name = publisher_name if publisher_name is not None else "Journal Owner"
+        # publisher_name = publisher_name if publisher_name is not None else "Journal Owner"
```

Comment:
yes, this is a better approach to this - having the default in the template now that it can include template logic.